protocol/protocol.go

```diff
@@ -133,21 +134,21 @@ type ParseUASTResponse struct {
 
 // Parser can parse UAST.
 type Parser interface {
-	ParseUAST(*ParseUASTRequest) *ParseUASTResponse
+	Parse(*ParseRequest) *ParseResponse
 }
 
-// DefaultParser is the default parser used by ParseAST and ParseUAST.
+// DefaultParser is the default parser used by Parse and ParseNative.
 var DefaultParser Parser
 
-// ParseUAST uses DefaultParser to process the given UAST parsing request.
+// Parse uses DefaultParser to process the given UAST parsing request.
```

Comment:
Update comment (`the given parsing request` or `the given parsing request to get the UAST`).